scripts/test_all_fuzzers.sh

```diff
@@ -67,7 +67,7 @@ do
 	    echo "[+] Done testing $fuzzer"
     else
         echo "[*] Building $fuzzer"
-        cargo build || exit 1
+        cargo build --target-dir ./target || exit 1
```

Comment:
Good call, but I don't think this is doing anything. (`--target-dir=target` is the default)
I've opened https://github.com/AFLplusplus/LibAFL/pull/845 to test shared target dirs. :)